src/modules/roles.py

```diff
@@ -1,8 +1,22 @@
 import discord
 from models import rank
 
-async def assignrole(ctx, guild, role: rank.Rank):
+async def assignrole(ctx, guild, role: rank.Rank, reason: str = None):
     r = discord.utils.get(guild.roles, name=str(role))
     if(r == None):
         return
-    await ctx.add_roles(r, reason="Auto-assiged by OLZBot")
+    if reason == None:
```

Comment:
```suggestion
    if(reason == None):
```
Mss wel handig om je bij de code conventions te houden